qfieldsync/__init__.py

```diff
@@ -25,9 +25,17 @@
 from __future__ import absolute_import
 
 import os
+import pathlib
+import sys
 
 import qgis.utils
 
+src_dir = pathlib.Path(__file__).parent.resolve()
+
+libqfieldsync_whl = src_dir / "libqfieldsync.whl"
```

Comment:
```suggestion
libqfieldsync_whl = src_dir.joinpath("libqfieldsync.whl")
```
I always feel afraid of too much magic with operator overload.